scripts/Text to handwritten/TextTohandwritten.py

```diff
@@ -0,0 +1,2 @@
+import pywhatkit
+pywhatkit.text_to_handwriting("Open source is a term that originally referred to open source software (OSS). Open source software is code that is designed to be publicly accessible—anyone can see, modify, and distribute the code as they see fit.",rgb=(0,0,225))
```

Comment:
Instead of hard coding a certain text, allow the user to specify a valid text file containing the text of their choice, then parse the text in the file and use the as input for your script.

Reply:
Also run the black linter locally for your lint tests to pass.